src/server/verification/processor/provider/ZoomProvider.js

```diff
@@ -111,17 +120,24 @@ class ZoomProvider implements IEnrollmentProvider {
         // so we'll reject with isNotMatch: true instead
         // to show the error screen on app side immediately
         // notifying about liveness check failed
-        throwCustomException(message, { isNotMatch }, response)
+        throwException(message, { isNotMatch }, response)
       }
 
+      resultBlob = get(response, 'scanResultBlob')
+
       // if liveness / security issues were detected
       if ([LivenessCheckFailed, SecurityCheckFailed].includes(name)) {
         isLive = false
 
         // notifying about liveness check failed
         await notifyProcessor({ isLive })
         log.warn(message, { enrollmentIdentifier })
-        throwCustomException(message, { isLive }, response)
+        throwException(message, { isLive, resultBlob }, response)
```

Comment:
why do we need the result blob?

Reply:
This is the breaking change from FaceTec. This field is used client side.
Please read the task related - there's a detailed description about it